modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -806,6 +807,10 @@ If you need to debug the import, it might be useful to collect the stack trace.
 This is done by using the `--verbose` option.
 |import.report
 
+|--schema=<path> footnote:[The optional `--schema` option is available in this version but not yet supported. It will be functional in a future release.]
```

Comment:
```suggestion
|--schema=<path> footnote:[The --schema option is available in this version but not yet supported. It will be functional in a future release.]
```